ffi/capi/src/bidi.rs

```diff
@@ -122,12 +122,16 @@ pub mod ffi {
         }
 
         /// The length of this map
-        #[allow(clippy::len_without_is_empty)]
         #[diplomat::attr(dart, rename = "length")]
         pub fn len(&self) -> usize {
             self.0.len()
         }
 
+        /// Whether this map is empty
+        pub fn is_empty(&self) -> bool {
```

Comment:
issue: I don't think we should have this over ffi, it's very much a Rust convention

Reply:
Using `foo.isEmpty()` instead of `foo.length() == 0` is a readability improvement that is mandated by many style guides and languages. This means we need to expose this in order to have idiomatic FFI.

Reply:
I guess, though we should probably be applying more renames then

(Not that we can pre-JS HIR)